src/opnsense/mvc/app/library/OPNsense/Auth/Voucher.php

```diff
@@ -134,6 +136,7 @@ private function userNameExists($username)
 
     private function setStartTime($username, $starttime)
     {
+        $this->dbHandle->exec("PRAGMA journal_mode=off");
```

Comment:
When writing to the database, SQLIte tries to create temporary file for logging in /conf/ He does not have to do this right. Or you need to give write permission on the directory /conf (which is not very good). Or deny to create a temporary file logging. You can still create the directory /conf/db is that SQLite will have rights to create temporary files.